src/vue/backend/src/templates/GuidedSetupContentCard.vue

```diff
@@ -84,7 +84,7 @@ export default {
     }
```

Comment:
The link button does not match design, in implementation it has blue color and a border/button color. In the design, it is black text